internal/config/config.go

```diff
@@ -0,0 +1,116 @@
+package config
+
+import (
+	"fmt"
+	"github.com/rs/zerolog"
+	"strings"
+	"sync"
+	"time"
+
+	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+	"github.com/spf13/viper"
+)
+
+type GlobalConfig struct {
+	Database struct {
+		Host        string
+		Port        uint16
+		Name        string
+		User        string
+		Password    string
+		MaxIdleTime time.Duration
+		MaxLifetime time.Duration
+		MaxOpenConn int
+		MaxIdleConn int
+		LogLevel    int
+	}
+	Logging struct {
+		Level  int
+		Stdout bool
+	}
+	Cloudwatch struct {
+		Enabled bool
+		Group   string
+		Stream  string
+	}
+	FeatureFlags struct {
+		Environment string
+	}
+	ExitOnPanic bool
+}
+
+var config GlobalConfig
+var configMutex sync.Mutex
```

Comment:
Just drop the mutex it is no longer necessary. This is cleaner.

Reply:
:+1: thanks